src/server.c

```diff
@@ -5283,6 +5283,17 @@ int processCommand(client *c) {
         return C_OK;
     }
 
+    /* check if DEBUG/Module command not allowed */
+    if ((server.debug_cmd_disabled && c->cmd->proc == debugCommand) ||
+        (server.module_cmd_disabled && c->cmd->flags & CMD_MODULE)) {
+        rejectCommandFormat(c,"%s commands not allowed, you may enable it by "
```

Comment:
```suggestion
        rejectCommandFormat(c,"%s command not allowed, you may enable it by "
```

p.s. the config you declared in the config file, said `*-commands-disabled` but your print here says `command` (no `s1), i changed the config name, so this print is ok.